src/App.tsx

```diff
@@ -15,71 +15,89 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [name, setName] = useState('Nothing');
+  const [clicked, setClicked] = useState(false);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function clear() {
+    setClicked(false);
+    const tbody: any = document.querySelectorAll('tbody');
+    const tbodyArray = [...tbody[0].childNodes];
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    for (let i = 0; i < tbodyArray.length; i += 1) {
+      const buttonClass = tbodyArray[i].childNodes[0].childNodes[0].className;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      if (buttonClass === 'button is-info') {
+        tbodyArray[i].childNodes[0].childNodes[0].className = 'button';
+        tbodyArray[i].childNodes[0].childNodes[0].childNodes[0].data = '+';
+      }
+    }
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    setName('Nothing');
+  }
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  function goodChoice(event: React.MouseEvent<HTMLButtonElement>) {
+    const copyEvent = event.currentTarget;
+
+    if (clicked === true) {
+      clear();
+      setClicked(false);
+    } else {
+      clear();
+      setName(event.currentTarget.parentNode?.nextSibling?.textContent || '');
+      copyEvent.textContent = '-';
+      copyEvent.className = 'button is-info';
+      setClicked(true);
+    }
+  }
+
+  return (
+
+    <main className="section container">
+      <h1 className="title">No goods selected</h1>
```

Comment:
![image](https://github.com/mate-academy/react_goods-selector/assets/86923089/482b87ee-6c70-494b-9802-89bd1a37800b)

Both titles are displayed
Use ternary operator to show one you need based on some condition